infra/terraform/modules/data_access/lambda_memberships.tf

```diff
@@ -5,8 +5,7 @@ resource "null_resource" "memberships_install_deps" {
     }
 
     triggers {
-        build_sh_sha = "${sha256(file("${path.module}/memberships/build.sh"))}"
-        requirements_sha = "${sha256(file("${path.module}/memberships/requirements.txt"))}"
+        force_rebuild = "${timestamp()}"
```

Comment:
This is a fix to something I noticed while working on it. Basically the building of the lambda function is not triggered on other devs machines because these `triggers`' attributes are kept in the TF remote store.

With hindsight I should have create a separate PR but I expected this PR to be merged sooner and that's why I included this small/kind-of-related fix here.